pkg/api/handlers/compat/networks.go

```diff
@@ -305,8 +306,9 @@ func RemoveNetwork(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "network not found", http.StatusNotFound, define.ErrNoSuchNetwork)
 		return
 	}
-	if err := network.RemoveNetwork(config, name); err != nil {
+	if err := network.RemoveNetwork(config, name, config.Engine.TmpDir); err != nil {
```

Comment:
Does this path make sense?  Should this be the cni path?  IE Could multiple containers create content by using different TmpDirs?

Reply:
this is using libpod's tmpdir to get a lock for all of cni.  at this point, containers dont matter.  the cni path is for where the plugins live on the filesystem so we know how/where to call them as they are not part of path.  the lock in question is "global, i need cni to not change for a second" lock.